net.seamly.seamly2d.yml

```diff
@@ -0,0 +1,55 @@
+app-id: net.seamly.seamly2d
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: seamly2d
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=cups
+  - --filesystem=home
+  - --device=dri
+rename-desktop-file: seamly2d.desktop
+separate-locales: false
+
+modules:
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_BOOST=OFF
```

Comment:
I'll assume this doesn't use the Splash backend of poppler.

Reply:
No, not that I am aware of